Projects/mnemos

```diff
@@ -0,0 +1,26 @@
+# mnemos
+
+## Name
+mnemos 
+
+## Summary
```

Comment:
It's totally okay to leave "TODO" comments here! I can either fix them before we merge this PR, or we can fix them up later.

Something like:

> Editors note: TODO - James fill this out!

or

> Editors note - James are there any other links we should include here?

are just fine.

You'll probably add a lot of these to start with! But that's okay! This will give me a bite-sized to-do list, which is much more manageable for me :)